lib/IRGen/GenDecl.cpp

```diff
@@ -1252,13 +1252,34 @@ void IRGenerator::noteUseOfTypeGlobals(NominalTypeDecl *type,
     return;
   
   // Force emission of ObjC protocol descriptors used by type refs.
-  if (auto proto = dyn_cast<ProtocolDecl>(type)) {
+  if (auto *proto = dyn_cast<ProtocolDecl>(type)) {
```

Comment:
Spurious change?

Reply:
Just a small thing I noticed. If it really bugs you I can remove it.

Reply:
It just doesn't seem like an improvement in any way.  Also, I have several times had to go through and fix a bunch of places where somebody needlessly recorded that something was a pointer only for a refactor to start encapsulating it in a class.

Reply:
Ok.